contributing/topics/documentation-standards.md

```diff
@@ -0,0 +1,64 @@
+# Provider Documentation Standards
+
+In an effort to keep the [provider documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs) consistent, this page documents some standards that have been agreed on. 
+
+This page will grow over time, and suggestions are welcome!
+
+## Notes
+
+Note blocks are used to provide additional information to users beyond the basic description of a resource, argument or attribute.
+
+In the past, there have been different approaches to how notes were formatted, some examples are:
+- Different words to indicate level of importance, e.g. `Info`, `Important`, `Caution`, and `Be Aware`.
+- Capitalisation differences, e.g. `Note:` vs `NOTE:`.
+- Whether or not a colon is included, e.g. `Note:` vs `Note`.
+
+Going forward, all notes should follow the exact same format (`(->|~>|!>) **Note:**`) where level of importance is indicated through the different types of notes as documented below. 
+
+<!-- 
+    - TODO: Considerations for when to add notes? We probably don't want to overdo it (More relevant to informational notes)
```

Comment:
Leaving a note here so I don't forget - we've been asked about how breaking changes should be documented. 

Whether the decision is for it to be added as a caution note or to allow a specific `!> ** Breaking Change:**` label for those cases, we should probably add a paragraph here to explain what the expectation is. We can discuss in the next team sync.